source/core/parameters.cc

```diff
@@ -1187,6 +1187,18 @@ namespace Parameters
                         "false",
                         Patterns::Bool(),
                         "Enable smoothing postprocessed vectors and scalars.");
+
+      prm.declare_entry(
+        "calculate VOF barycenter",
+        "false",
+        Patterns::Bool(),
+        "Enable calculation of the barycenter location and velocity of the monitored fluid in VOF simulations.");
```

Comment:
```suggestion
        "Enable calculation of the barycenter location and velocity of the fluid 1 in VOF simulations.");
```

To be consistent with the doc